lib/src/rules/always_specify_types.dart

```diff
@@ -102,6 +104,18 @@ class AlwaysSpecifyTypes extends LintRule implements NodeLintRule {
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
+  static const LintCode keywordCouldBeTypeCode = LintCode(
+      "always_specify_types", "'{0}' could be '{1}'.",
```

Comment:
The problem message should indicate what's wrong. The problem isn't that the keyword could be a type, but that there is no type annotation when one is required. It would be better if the problem message were something like "Missing a type annotation.".

On the other hand, the correction message should indicate what to do to fix the problem, and that's a great place to include one possible type. Perhaps something like "Try replacing '{0}' with '{1}'."

Similarly with the other error codes.